### PR TITLE
Loader with  saving text

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes
image

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

**Review by alanraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Item/View.js [hunk 1]
```diff
@@ -170,7 +174,24 @@ const View = React.memo((props) => {
   const saveDraft = (payload) => {
     let dataChanged = !isEqual(payload.data, lastUpdatedDraft.data);
     if (draftSubmissionId && isDraftCreated) {
-      if (dataChanged) dispatch(draftUpdateMethod(payload, draftSubmissionId));
+      if (dataChanged) {
+        setDraftCreating(true);
+        dispatch(
+          draftUpdateMethod(payload, draftSubmissionId, (err) => {
+            if (err) {
+              toast.error(err);
```
Comment (sreehari-aot) [inline forms-flow-web/src/components/Form/Item/View.js:182]: This is a bad indicator.

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/Item/View.js:182]: Remove Toast for Draft . Will find some better alternative say offline  Case

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-web/src/components/Draft/Edit.js [hunk 2]
```diff
@@ -83,7 +84,22 @@ const View = React.memo((props) => {
...
+        setDraftCreating(true);
+        dispatch(
+          draftUpdate(payload, draftSubmission?.id, (err) => {
+            if (!err){
+              setTimeout(() => {
+                setDraftSaved(true);
+                setTimeout(() => {
+                  setDraftCreating(false);
+                  setDraftSaved(false);
+                }, 2000);
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Draft/Edit.js:97]: isnt the timer coming from Environment  ???

Comment (abhilash-aot) [reply forms-flow-web/src/components/Draft/Edit.js:97]: @sreehari-aot / @shuhaib-aot

**Review by abhilash-aot — CHANGES_REQUESTED** [review]

**Review by abhilash-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**